src/test/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailServiceTest.java

```diff
@@ -95,6 +95,8 @@ void sendObjectionSubmittedCustomerEmail() throws ServiceException {
         assertTrue(data.containsValue("Company: " + COMPANY_NUMBER));
         assertTrue(data.containsValue(OBJECTION_ID));
         assertTrue(data.containsValue(attachments));
+        assertTrue(data.containsValue(EMAIL));
+        assertTrue(data.containsKey("subject"));
```

Comment:
I think you need a test for the value also, checking that the formatting/string replacement has worked as expected.